source/src/molecular-unfolding/cdk/resources/onStart.template

```diff
@@ -24,6 +24,14 @@ pip3 install jupyter ipykernel
 
 python3 -m ipykernel install --user --name=qcenv
 
+aws s3 cp s3://aws-gcr-solutions/quantum-computing-exploration-for-drug-discovery-on-aws/__VERSION__/code.zip /home/ec2-user/SageMaker/ || true
```

Comment:
how to publish the `code.zip` to that bucket? what's the process?

Reply:
I prefer to bundling a S3 asset when building/publishing the project.

see https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3_assets-readme.html#asset-bundling for how to archive it.